hypervisor/arch/x86/guest/vmcall.c

```diff
@@ -233,6 +233,7 @@ int32_t vmcall_vmexit_handler(struct acrn_vcpu *vcpu)
 
 	if (!is_hypercall_from_ring0()) {
 		pr_err("hypercall is only allowed from RING-0!\n");
+		vcpu_inject_gp(vcpu, 0U);
```

Comment:
if we return -EACCES, vcpu_thread side will inject #GP too. Do we need this one?